### ISSUE TITLE
Smoke tests should not output to `os.Stdout` or `os.Stderr` when successful

### ISSUE THREAD

**Issue by selzoc** [cloudfoundry-incubator/routing-acceptance-tests#7]
Hello!

I noticed since we run `rats` a lot that the smoke tests portion has output like:
```
Running Suite: SmokeTests Suite
===============================
Random Seed: 1507334986 - Will randomize all specs
Will run 1 of 1 specs


Connecting to URL http://tcp.snitch.cf-app.com:1035... 

Received response 200

Connecting to URL tcp.snitch.cf-app.com:1035... 

Received error while connecting dial tcp 104.197.81.219:1035: getsockopt: connection refused
•
Ran 1 of 1 Specs in 85.893 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 0 Skipped PASS
```
(from https://release-integration.ci.cf-app.com/teams/main/pipelines/cf-deployment/jobs/lite-rats/builds/113)

As you can see, even though the test passed, it still had output outside of the standard `ginkgo` metadata.  Poking around at the code, I see that `os.Stdout` and `os.Stderr` are used as an argument to `fmt.Fprintf` in several places.  If instead `GinkgoWriter` was used, the output would be hidden on success unless you set the `verbose` flag:
```
Ginkgo provides a globally available io.Writer called GinkgoWriter that you can write to. GinkgoWriter aggregates input while a test is running and only dumps it to stdout if the test fails. When running in verbose mode (ginkgo -v or go test -ginkgo.v) GinkgoWriter always immediately redirects its input to stdout.
```

Note that in `http_routes` suite, `GinkgoWriter` is used.

**Comment by aaronshurley** [comment]
This fix was included as part of [routing-release v0.167.0](https://github.com/cloudfoundry-incubator/routing-release/releases/tag/0.167.0). Please let us know if you have further concerns.

**Comment by selzoc** [comment]
yay!  thanks much 💯
